applicationinsights.json

```diff
@@ -1,6 +1,6 @@
 {
   "role": {
-    "name": "hmpps-nomis-visits-mapping-service"
+    "name": "nomis-visits-mapping-service"
   },
```

Comment:
not sure what role does, but assuming that is the cloudRoleName this will be inconsistent with our other services that have `HMPPS` and the front, so it might lead to people being frustrated in why they can't find any appinsight stuff

Reply:
ok, will revert